src/libs/ajax/data-table-providers/WDSDataTableProvider.test.ts

```diff
@@ -0,0 +1,165 @@
+import { EntityMetadata, EntityQueryOptions, EntityQueryResponse } from './DataTableProvider'
+import { RecordQueryResponse, RecordTypeSchema, WDSDataTableProvider } from './WDSDataTableProvider'
+
+
+const uuid = '123e4567-e89b-12d3-a456-426614174000' // value doesn't matter for these tests
+
+const provider = new WDSDataTableProvider(uuid)
```

Comment:
a provider should be instantiated inside each unit test to avoid potential cross-test state creep in the class under test.